src/main/resources/ListedElectronicMonograph.json

```diff
@@ -68,11 +68,16 @@
           }
         },
         {
-          "field": "FUND_CODE",
+          "field": "FUND_ID",
           "dataSource": {
             "from": "//FundCode",
-            "default": "0",
-            "translation": "lookupMock"
+            "translation": "lookupFundId"
```

Comment:
In all the existing mappings, if a specified type is not found we are falling back to the first one listed from the API, I think we need to take the same approach here. Please assign a default value of * and pick the first record, if fund ID is not found.

Reply:
implementation updated, added test to exercise this case

Reply:
I'm not crazy about this approach but it is consistent with other lookups.  We may need to eventually come up with something better.  Also note:

* This would only happen if GOBI's configuration is out of sync with FOLIOs (e.g. a fund exists in GOBI, but not in FOLIO.  
* Eventually we'll have tenant-specific mappings defined - in this case a default fund could be specified, e.g. a "catch-all" or "general" fund